src/menus/DirektverbindungenMenu/DvMenu.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-param-reassign */
+import React, { useEffect, useMemo } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { makeStyles } from '@material-ui/core';
+import DvFeatureInfo from '../../config/ch.sbb.direktverbindungen/DvFeatureInfo';
+import DvLayerSwitcher from './DvLayerSwitcher';
+import { setDisplayMenu } from '../../model/app/actions';
+import IframeMenu from '../IframeMenu';
+import FadeShadow from '../../components/FadeShadow';
+
+const useStyles = makeStyles(() => {
+  return {
+    wrapper: {
+      padding: '15px 10px',
+    },
+  };
+});
+
+function DvMenu() {
+  const dispatch = useDispatch();
+  const classes = useStyles();
+  const screenWidth = useSelector((state) => state.app.screenWidth);
+  const featureInfo = useSelector((state) => state.app.featureInfo);
+  const topic = useSelector((state) => state.app.activeTopic);
+  const isMobile = useMemo(() => {
+    return ['xs'].includes(screenWidth);
+  }, [screenWidth]);
+
+  const urlSearch = useMemo(
+    () => new URLSearchParams(window.location.search),
+    [],
+  );
+
+  const switcher = useMemo(
+    () => (
+      <div className={classes.wrapper}>
+        <DvLayerSwitcher row={isMobile} />
```

Comment:
this `row` property is useless, you use it just for a creepy css . I don't know how but you can get rid of this , modifying the parent element css.

Reply:
This is a prop in [MUI FormGroup](https://v4.mui.com/api/form-group/#formgroup-api), I don't only use it for creepy CSS, but also [here](https://github.com/geops/trafimage-maps/pull/1075/files/1092cc677bb098a82ae801b08804ef98966eac9e#diff-9affd86c0f0166e6fb1ff0e1e70a2cd3235bd53c817b02496543b49f1f9da0b0R99).

Reply:
ok but you can still fix the creepy css   top: 4px

Reply:
I think we need this, it's MUI